docker/zap-api-scan.py

```diff
@@ -152,7 +152,7 @@ def main(argv):
     fail_inprog_count = 0
 
     try:
-        opts, args = getopt.getopt(argv, "t:f:c:u:g:m:n:r:J:w:x:l:hdaijp:sz:P:D:T:O:", ["hook="])
+        opts, args = getopt.getopt(argv, "t:f:c:u:g:m:n:r:J:w:x:l:hdaijp:sz:P:D:S:T:O:", ["hook="])
```

Comment:
Have you actually tried this out?
The additional colon will mean that it will require another parameter, so I think this will fail.

Reply:
My bad , was to eager to fix this ASAP.